maven-surefire-plugin/src/site/apt/index.apt.vm

```diff
@@ -141,6 +141,8 @@ mvn verify
 
   * {{{./examples/testng.html}Using TestNG}}
 
+  * {{{./examples/junit-platform.html}Using JUnit Platform}}
```

Comment:
Not sure if users would realize this is JUnit5 provider.

Reply:
Can we just mention `Using JUnit5 Platform` - added `5`.

Reply:
Sure.